mirrulations-dashboard/src/mirrdash/static/index.js

```diff
@@ -70,11 +91,13 @@ const updateClientDashboardData = () => {
         } = jobInformation;
         updateHtmlValues('jobs-waiting', num_jobs_waiting, jobs_total);
         updateHtmlValues('jobs-done', num_jobs_done, jobs_total);
+        updateCorpusProgressHtml([num_dockets_done, num_documents_done, num_comments_done], [232255, 1718669, 18072106]); //TO DO: change hard coded numbers
```

Comment:
Are you guys doing the attachments estimate as well? On my system I'm getting NAN for the attachments percentage.

Reply:
I see the TODO now, if it helps I found there were about at least 15 million attachments from a screenshot earlier in the semester